### PR TITLE
20 create 1 pager edit page

### PR DESCRIPTION
**Pull request by binnmti**
image

## Summary by CodeRabbit

- **New Features**
	- Added markdown support in the document editing page, enabling users to input and render markdown content.
	- Introduced input fields for document title, status, authors, reviewers, and last updated date on the editing page.

- **Style**
	- Implemented new styling for the document editing page to enhance user interface and experience.

### CODE REVIEW

File: one-pager-maker/src/pages/Edit.tsx [hunk 1]
```diff
@@ -1,10 +1,51 @@
...
     <div>
-      <h1>Edit</h1>
+      <div className="document-title-div">
+        <input className="document-title" type="text"></input>
+        <button type="button">Save</button>
+      </div>
+      <div className="document-parameter-div">
+        <span className='document-parameter-left'>
+            <select name="status">
+              <option value="draft" selected>draft</option>
```
[the remark this answers is not in the document]
Comment (binnmti) [reply one-pager-maker/src/pages/Edit.tsx]: fixed

File: .codeclimate.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 exclude_patterns:
   - "one-pager-maker/src/redux/document/documentsApi.ts"
   - "one-pager-maker/tests/integration/redux/documentsApi.test.tsx"
+
+checks:
```
Comment (binnmti) [inline .codeclimate.yml:5]: CodeClimate Change Rule

File: one-pager-maker/src/pages/Edit.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.document-parameter-div {
+  display: flex;
+  justify-content: space-between;  
+  border: solid 1px;
+}
+
+.document-parameter-left {
+  text-align: left;
+  float: left; 
+}
```
Comment (Harineko0) [inline one-pager-maker/src/pages/Edit.css:20]: gap between each input might be needed
```
display: flex;
gap: 12px
```

File: one-pager-maker/src/pages/Edit.tsx [hunk 4]
```diff
@@ -1,10 +1,51 @@
...
+      <div className="document-parameter-div">
+        <span className='document-parameter-left'>
+            <select name="status">
+              <option value="draft">draft</option>
+              <option value="reviewed">reviewed</option>
+              <option value="final">final</option>
+              <option value="obsolete">obsolete</option>
+            </select>
+            <span>
+              <input className="authors" type="text"></input>
```
Comment (Harineko0) [inline one-pager-maker/src/pages/Edit.tsx:26]: You can use [mui textfield](https://mui.com/material-ui/react-text-field/) but pure html is not bad

**Review by Harineko0 — APPROVED** [review]
LGTM!

**Pull request merged**